### PR TITLE
Zero length matches

### LINKED ISSUE #12: Typing ".*" in crashes the demo
In general, it's pretty surprising that input is interpreted as a regex by default, so that "." highlights all characters. But it seems particularly bad that ".*" crashes the highlighter unless you specifically turn on `autoEscape`. This is reproducible at https://bvaughn.github.io/react-highlight-words/

I would consider defaulting `autoEscape` on, but I don't want to overprescribe a solution to the bug.

### PR DESCRIPTION
**Pull request by RichardBradley**
This fixes https://github.com/bvaughn/react-highlight-words/issues/12

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "lint": "standard",
     "prebuild": "rimraf dist",
     "prepublish": "npm run build",
-    "test": "mocha --compilers js:babel-register 'src/**/*.test.js'"
+    "test": "mocha --compilers js:babel-register \"src/**/*.test.js\""
```
Comment (RichardBradley) [inline package.json:13]: (Fix test pattern for Windows users, see mochajs/mocha#2895 )

**Comment by RichardBradley** [comment]
Do I need to make any changes to  bvaughn/react-highlight-words to pick up a new version?

File: package.json (hunk 1 above)
Comment (bvaughn) [reply package.json:13]: Nice!

**Review by bvaughn — APPROVED** [review]
Thanks! 😄

**Comment by bvaughn** [comment]
> Do I need to make any changes to bvaughn/react-highlight-words to pick up a new version?

No, you shouldn't need to since this can go into a bugfix release. It should just be picked up by default.

**Comment by bvaughn** [comment]
Published via 1.1.2

**Comment by bvaughn** [comment]
Just bumped the demo to use this version

**Comment by RichardBradley** [comment]
Thanks

**Pull request merged**